src/App.js

```diff
@@ -1,5 +1,37 @@
+import Racingcar from './Racingcar.js';
+import ioMethod from './inputoutputMethod.js';
+import MAGICNUMBER from './magicnumber.js';
+import ERRORMESAGE from './errorMessage.js';
+import validation from './validation.js';
+import utils from './utils.js';
+
 class App {
-  async run() {}
+  #racingcar;
+  #tryNum;
+  #board;
+
+  async run() {
+    await this.#gameSetting();
+    this.#carRacing();
+    ioMethod.printFirstPlayer(this.#racingcar.rank(this.#board));
+  }
+
+  async #gameSetting() {
+    const inputs = await ioMethod.inputCarName();
+    const inputArr = inputs.split(MAGICNUMBER.SEPARATOR);
+    this.#racingcar = new Racingcar(inputArr);
+    this.#tryNum = await ioMethod.inputTryNum();
+    this.#tryNum = utils.tryNumvalidation(this.#tryNum);
+    this.#board = utils.createBoard(inputArr);
+    ioMethod.printProgressResult();
+  }
```

Comment:
```suggestion
  async #gameSetting() {
    const inputs = await ioMethod.inputCarName();
    const inputArr = inputs.split(MAGICNUMBER.SEPARATOR);

    this.#tryNum = await ioMethod.inputTryNum();
    this.#tryNum = utils.tryNumvalidation(this.#tryNum);

    this.#racingcar = new Racingcar(inputArr);

    this.#board = utils.createBoard(inputArr);
    ioMethod.printProgressResult();
  }
```

게임 세팅은 입력, 레이싱카 생성, 보드 생성, 출력 등의 역할을 담당하고 있는거로 보여요! 

이렇게 문맥이 존재하고 구분된다면 가독성과 이해를 위해 비슷한 역할을 담당하는 코드별로 묶어서 구분해주는 것도 좋을 거 같아요!